lib/unleash/strategy/flexible_rollout.rb

```diff
@@ -41,6 +42,8 @@ def resolve_stickiness(stickiness, context)
         when 'random'
           random
         when 'default'
+          return random unless context.instance_of?(Unleash::Context)
+
           context.user_id || context.session_id || random
```

Comment:
Don't think we need the line above, we should be able to do

```suggestion
   context&.user_id || context&.session_id || random
```

Reply:
Given that resolve_stickiness is a private method, I agree that it is not the correct place to check that the argument is not of the correct type. It should be a safe assumption. @sighphyre 's suggestion is a good one. Using the `&. ` (safe navigation operator) is always healthy.

Reply:
ok, fixed